object/types.proto

```diff
@@ -135,6 +135,13 @@ message Header {
   //   Human-friendly name
   // * FileName \
   //   File name to be associated with the object on saving
+  // * FilePath \
+  //   Full path to be associated with the object on saving. Should start with a
+  //   '/' and use '/' as delimiting symbol. Trailing '/' should be interpreted
```

Comment:
Why we want to have such limitation? In S3 we can create both `/some/path/to/object` and `some/path/to/object`

Reply:
It's a recommendation to simplify directory hierarchy construction and avoid problems with incorrect path mapping. For example, a typical http proxy rewrite rules block expects object path starting with '/'. Otherwise, it needs to try both possible variants. We want to avoid such situations.